### ISSUE TITLE
BUG - 2 - iOS/Android - Accessibility: Navigating from Payment Breakdown Modal to Payment History page does not read title of page

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#10772]
## What happened?
When using the screen reader on iOS/Android devices, when the user navigates from the Payment Breakdown modal to the Payment History page, the screen reader does _not_ read out that the user is on the Payment History page.

## Specs:

- Device: Samsung Galaxy S24, iPhone 15 plus.
- OS: Latest.
- User Account: Demo
- Accessibility State: Screen reader ON, default accessibility options.
- App build: 2.48.1 (2645)

## Steps to Reproduce

1. In the app on your device, log in as the DEMO user.
2. While the screen reader ON for the device, navigate down the Home page to the Latest Payment section.
3. Select the **See details** button in the **Latest payment** section.
4. While the modal is open, select the **Go to payment history** link.
5. Observe - When the screen reader navigates to the Payment History page, the screen reader does **not** read out that the user is on the Payment history page.
 
## Desired behavior

While the screen reader is on, when the use navigates to the Payment History page from the payment modal, the screen reader should announce that the user is on the Payment History page.

## Acceptance Criteria

While the screen reader is on, when the use navigates to the Payment History page from the payment modal, the screen reader should focus on the header of the Payment History page and announce that the user is on that page.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: High 

- 3 - Low
**- 2 - High**
- 1 - Critical

## Linked to Story

#10748 

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/66f641ff-f6a5-4f80-b386-fd3125e725ab

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  
N/A
</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by keli13** [comment]
Like 10780, this is related to how the app is handling focus when accessibility is turned on. We can lump this together as a ticket/task to see if we can fix how we handle focus in general when navigating between pages.

**Comment by mmutia** [comment]
Hi all,

Per internal discussion with the ADE team in regard to sending focus to the header vs the back button upon loading the Payment History page, we recommend that focus is sent to the header instead of the back button. We have decided to make this recommendation based on the following reasons:

1. While AT may announce the labeled button to access the Payment History page (along with a brief activation noise), the user may have difficulty understanding that the page itself has loaded without the announcement of the heading. With only the announcement of the labeled button and the activation noise, the user will only be certain that the button has been activated -- not that the page itself has loaded.
2. Native gestures are built into AT to return the user back to the previous screen. While custom gestures can be set for this purpose, native gestures are commonly used to exit screens.
3. If the concern for locating the back button persists (and while we do know that setting the back button directly under the heading may not be good app design), is it possible that the back button can be located directly after the heading in the focus order? This may solve the potential problem of users being unable to locate the back button while being unaware of the native gestures to exit the current screen.
4. General orientation and consistency for the user.

Please let us know if you have any questions or concerns!

**Comment by digitalken** [comment]
@matthew-df 
This one is fixed on iOS, but having a problem with Android.  The Samsung will focus on the "What if I'm missing a payment?" link.  It also doesn't announce the link and says "VA".  The Pixel will focus on the back to payments link.  Neither of my Androids focuses on the Header.  The video is a Samsung.  I took a screen shot and circled what it is focusing on with the two Androids.

<img width="536" height="547" alt="Image" src="https://github.com/user-attachments/assets/388bde93-c224-41b0-ac29-6a61ad4934d6" />

https://github.com/user-attachments/assets/04f45071-3f2b-4be2-867e-ea626c724e78
